luigi/cmdline.py

```diff
@@ -19,7 +19,7 @@ def luigid(argv=sys.argv[1:]):
     parser.add_argument(u'--background', help=u'Run in background mode', action='store_true')
     parser.add_argument(u'--pidfile', help=u'Write pidfile')
     parser.add_argument(u'--logdir', help=u'log directory')
-    parser.add_argument(u'--state-path', help=u'Pickled state file')
+    parser.add_argument(u'--state_path', help=u'Pickled state file')
```

Comment:
Let's aim to change documentation not code. It looks like this change directly contradicts the line I first commented on.